package/Dockerfile

```diff
@@ -66,32 +75,36 @@
     meson compile -C .build && \
     meson install -C .build --destdir /usr/local/nvme
 
-FROM registry.suse.com/bci/bci-base:15.4 AS release
+# Install grpc_health_probe
+RUN wget https://github.com/grpc-ecosystem/grpc-health-probe/releases/download/v0.3.2/grpc_health_probe-linux-${ARCH} -O /usr/local/bin/grpc_health_probe && \
+    chmod +x /usr/local/bin/grpc_health_probe
+
+FROM registry.suse.com/bci/bci-base:15.5 AS release
 
 ARG ARCH=amd64
 
 RUN zypper -n addrepo --refresh https://download.opensuse.org/repositories/system:/snappy/SLE_15/system:snappy.repo && \
     zypper -n addrepo --refresh https://download.opensuse.org/repositories/network:/utilities/SLE_15_SP4/network:utilities.repo && \
-    zypper -n addrepo --refresh https://download.opensuse.org/repositories/devel:libraries:c_c++/15.4/devel:libraries:c_c++.repo && \
-    zypper -n addrepo --refresh https://download.opensuse.org/repositories/devel:languages:python:Factory/15.4/devel:languages:python:Factory.repo && \
+    zypper -n addrepo --refresh https://download.opensuse.org/repositories/devel:libraries:c_c++/15.5/devel:libraries:c_c++.repo && \
+    zypper -n addrepo --refresh https://download.opensuse.org/repositories/devel:languages:python:Factory/15.5/devel:languages:python:Factory.repo && \
     zypper -n addrepo --refresh https://download.opensuse.org/repositories/devel:languages:python:backports/SLE_15/devel:languages:python:backports.repo && \
     zypper --gpg-auto-import-keys ref
 
-RUN zypper -n install kmod curl fuse wget \
-    nfs-client nfs4-acl-tools cifs-utils \
-    libibverbs librdmacm1 rdma-core-devel perl-Config-General libaio-devel sg3_utils \
-    iputils telnet iproute2 qemu-tools \
-    e2fsprogs e2fsprogs-devel xfsprogs xfsprogs-devel util-linux-systemd python3
+
+RUN zypper -n install nfs-client nfs4-acl-tools cifs-utils sg3_utils \
+    iputils iproute2 qemu-tools e2fsprogs util-linux-systemd python3-pyelftools libcmocka-devel&& \
+    rm -rf /var/cache/zypp/*
 
 # Install SPDK dependencies
 COPY --from=builder /usr/src/spdk/scripts /spdk/
-RUN if [ "${ARCH}" == "amd64" ]; then \
-        sh /spdk/pkgdep.sh; \
-        rm -rf /tmp/spdk; \
+RUN if [ "${ARCH}" = "amd64" ] || [ "${ARCH}" = "arm64" ]; then \
```

Comment:
If both are supported, let's just remove this if statement.

Reply:
We won't support s390x anymore, right?

Reply:
Yeah. We are going to remove s390.